test/test_nn.py

```diff
@@ -10532,6 +10540,27 @@ def test_conv_cudnn_mismatch_memory_format(self, device):
             self._test_conv_cudnn_nhwc_nchw(nn.Conv2d, n, c, h, w, k, filter_size, device)
             self._test_conv_cudnn_nhwc_nchw(nn.ConvTranspose2d, n, c, h, w, k, filter_size, device)
 
+    @onlyCUDA
```

Comment:
I think we need to clearly identify cases when usage of `convert_conv2d_weight_memory_layout` is better than doing `model = model.to(memory_format=torch.channels_last)` as this particular case not covering it.

Reply:
Do we just want something that explains what `convert_conv2d_weight_memory_layout` does?

I'm copying the message from slack over:
> Calling `model.to(memory_format=torch.channels_last)` is more aggressive.Layers that has 4d tensor as weight may or may not get speedup converting weights to NHWC. One thing we (relatively) know is that NHWC transpose for convolution is beneficial and hopefully will justify the transpose if it is actually required.
> Outside of this context, I don't know if there is another layer with 4d weight that would be affected by .to (a pointwise layer with a 4d weight would be beneficial to be in the same memory_format as the input is.)

I'll get a cleaner text and throw them under the docString of `convert_conv2d_weight_memory_layout`.

Reply:
Looking forward for cleaner text. Everything else looks good for me.